src/Logic/DirectionalLight.java

```diff
@@ -0,0 +1,38 @@
+package Logic;
+
+public class DirectionalLight extends Light
+{
+	public DirectionalLight()
+	{
+		super();
+	}
+	
+	public DirectionalLight(float directionX, float directionY, float directionZ,  float colorR, float colorG, float colorB)
+	{
+		super(colorR, colorG, colorB);
+		this.directionX = directionX;
+		this.directionY = directionY;
+		this.directionZ = directionZ;
+	}
+	
+	public float directionX = 0;
+	public float directionY = 0;
+	public float directionZ = 0;
+	
+	void normalizeData()
```

Comment:
[Fix] de ce este aceasta metoda default? iar asta pare a fi logica de business, nu are ce cauta in entitate

Reply:
l-am lasat acolo pentru ca mi s-a parut mai ok sa scriu l.normalize() decat LightNormalizer.normalize(l); nu prea vad ce sens ar avea o clasa de genul atata timp cat nu mentine un state intern